pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -226,18 +239,51 @@ func triggerSnapshots(ctx *zedmanagerContext, status *types.AppInstanceStatus) {
 	publishAppInstanceStatus(ctx, status)
 }
 
-func triggerRollback(ctx *zedmanagerContext, status types.AppInstanceStatus) error {
+func triggerRollback(ctx *zedmanagerContext, status *types.AppInstanceStatus) (*types.AppInstanceConfig, error) {
```

Comment:
Would make sense to document that this returns the appinstanceconfig which was previously snapshotted?